nmdc_runtime/api/models/wfe_file_stages.py

```diff
@@ -23,3 +25,41 @@ class GlobusTask(BaseModel):
     task_status: str = Field(
         ..., description="Status of the Globus task.", examples=["Some status"]
     )
+
+
+class JGISample(BaseModel):
+    """
+    Represents a JGI Sample for workflow file staging.
+    """
+
+    jdp_file_id: str = Field(
+        ..., description="JDP File ID", examples=["Some JDP File ID"]
+    )
+    ap_gold_id: str = Field(..., description="AP Gold ID", examples=["Some AP Gold ID"])
+    study_id: str = Field(..., description="Study ID", examples=["Some Study ID"])
+    its_ap_id: int = Field(..., description="ITS AP ID", examples=[12345])
+    project_name: str = Field(
+        ..., description="Project Name", examples=["Some Project Name"]
+    )
+    biosample_id: str = Field(
+        ..., description="Biosample ID", examples=["Some Biosample ID"]
+    )
+    seq_id: str = Field(..., description="Sequence ID", examples=["Some Sequence ID"])
+    file_name: str = Field(..., description="File Name", examples=["some_file.fastq"])
+    file_status: str = Field(
+        ..., description="File Status", examples=["Some File Status"]
+    )
+    file_size: int = Field(..., description="File Size", examples=[123456])
+    md5sum: Optional[str] = Field(
+        None, description="MD5 Sum", examples=["Some MD5 Sum"]
```

Comment:
This is an example MD5 sum from [Wikipedia](https://en.wikipedia.org/wiki/Md5sum#Check_single_MD5).

```suggestion
        None, description="MD5 Sum", examples=["D43F2404CA13E22594E5C8B04D3BBB81"]
```